platforms/core-configuration/configuration-cache/src/main/kotlin/org/gradle/internal/cc/impl/CheckedFingerprint.kt

```diff
@@ -27,13 +27,14 @@ sealed class CheckedFingerprint {
     // Everything is up-to-date
     object Valid : CheckedFingerprint()
 
-    data class Found(val entryName: String) : CheckedFingerprint()
+    class Found(val entryName: String) : CheckedFingerprint()
 
     // The entry cannot be reused at all and should be recreated from scratch
     class EntryInvalid(val buildPath: Path, val reason: StructuredMessage) : CheckedFingerprint()
 
     // The entry can be reused, however the values for certain projects cannot be reused and should be recreated
     class ProjectsInvalid(
+        val entryName: String,
```

Comment:
Not a big fan of this particular change since it forces us to pass `candidateEntry` to the downstream of project fingerprint check. 

I'd say that CheckedFingerprint probably requires some remodelling, since currently it's a mix of entire fingerprint checking results (`NotFound` | `Valid`) and individual entry checking result (`EntryInvalid`|`ProjectsInvalid`|`Found`)